js/index.js

```diff
@@ -28,7 +28,11 @@ const cards = [
 const memoryGame = new MemoryGame(cards);
 
 window.addEventListener('load', (event) => {
+  let card1 = '';
```

Comment:
I think variables such as cards should be inside the MemoryGame class which defines the logic of the game